ably/rest/push.py

```diff
@@ -142,6 +144,18 @@ def list(self, **params):
             self.ably.http, url=path,
             response_processor=channel_subscriptions_response_processor)
 
+    def list_channels(self, **params):
+        """Returns a PaginatedResult object with the list of
+        PushChannelSubscription objects, filtered by the given parameters.
+
+        :Parameters:
+        - `**params`: the parameters used to filter the list
+        """
+        path = '/push/channels' + format_params(params)
+        response_processor = channels_response_processor
```

Comment:
Just curious. Couldn't you pass `channels_response_processor` to `paginated_query( ... )` and skip this assignment?